### PR TITLE
Integrate skeleton

### PR DESCRIPTION
**Pull request by tokland**
Closes ch22 

(https://app.clubhouse.io/eyeseetea/story/22/skeleton-app-update)

### CODE REVIEW

**Comment by ifoche** [comment]
[ch22]

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+SKIP_PREFLIGHT_CHECK=true
+PORT=8081
+BROWSER=false
+REACT_APP_DHIS2_BASE_URL=http://dev2.eyeseetea.com:8082
```
Comment (adrianq) [inline .env]: Not sure about this... either we remove it or we point to 8085

File: i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+msgstr ""
+"Project-Id-Version: i18next-conv\n"
+"POT-Creation-Date: 2019-10-10T10:41:21.864Z\n"
+"PO-Revision-Date: 2018-10-25T09:02:35.143Z\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+
+msgid "Click to show feedback"
```
Comment (adrianq) [inline i18n/es.po]: Is Spanish required? If so why there are so few translations in comparison with English

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+        <title>React App</title>
```
Comment (adrianq) [inline public/index.html]: Gee App?

File: .env (hunk 1 above)
Comment (tokland) [reply .env]: Let's remove it.

File: i18n/es.po (hunk 2 above)
Comment (tokland) [reply i18n/es.po]: Not sure if it's required, they can be easily added/revoved, but there was a fix to make in the calls to update-po task.

**Pull request merged**